uinputwrapper.go

```diff
@@ -317,6 +375,26 @@ func syncEvents(deviceFile *os.File) (err error) {
 	return err
 }
 
+func sendEvent(deviceFile *os.File, eventType uint16, eventCode uint16, eventValue int32) error {
+	iev := inputEvent{
+		Time:  syscall.Timeval{Sec: 0, Usec: 0},
+		Type:  eventType,
+		Code:  eventCode,
+		Value: eventValue}
+
+	buf, err := inputEventToBuffer(iev)
+	if err != nil {
+		return fmt.Errorf("writing abs event failed: %v", err)
+	}
+
+	_, err = deviceFile.Write(buf)
+	if err != nil {
+		return fmt.Errorf("failed to write rel event to device file: %v", err)
```

Comment:
Is this really a rel or rather an abs event? If I understand this correctly, the sendEvent function is currently only used for abs events. Therefore, this message is misleading.